modules/social_features/social_profile/modules/social_profile_manager_notes/css/manager_notes.css

```diff
@@ -0,0 +1,36 @@
+.comment-comment-profile-form {
+  margin-top: 10px;
```

Comment:
You werent able to remove the CSS and use classes and alters to do the trick?

Reply:
we should be using this file form--comment.html.twig to render the form
As with Polls I had troubles with it, so I kept the CSS.
If you do it please assign me as reviewer so that I can learn how to do it :)